builtins: allow full consistency check w/o nuking cluster

tbg opened the pull request:
Previously, triggering a full consistency check would have potential to
harm the cluster due to triggering checks with full DistSender
parallelism. Limit to one check at a time for a (much slower, but) less
harmful check (that we can potentially emit in roachtests).

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/41632)
<!-- Reviewable:end -->

tbg committed:
builtins: allow full consistency check w/o nuking cluster

Previously, triggering a full consistency check would have potential to
harm the cluster due to triggering checks with full DistSender
parallelism. Limit to one check at a time for a (much slower, but) less
harmful check (that we can potentially emit in roachtests).

Release note: None

andreimatei reviewed (commented):
LGTM

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/41632#-:-LrK0Q8s1z_NL7UhsBk9:bb6ywmx)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @andreimatei)



<!-- Sent from Reviewable.io -->
